### PR TITLE
Make verbose logging output optional on Docker 

### PR DESCRIPTION
**Pull request by TheFlyingBadger**
make "--verbose" optional based on ONEDRIVE_VERBOSE environment variable. Verbose will continue to be the default if ONEDRIVE_VERBOSE is set to *anything* other than 0 (zero), or if it's not set at all.

### CODE REVIEW

**Comment by norbusan** [comment]
Hi @TheFlyingBadger 
that looks fine to me, could you try to add a few words to `docs/DOCKER.md` so that is documented outside the code?

Thanks

**Comment by TheFlyingBadger** [comment]
Not a problem. I'll also document ONEDRIVE_UID and ONEDRIVE_GID in there
too (i needed to look in the code to find it!)

**Comment by norbusan** [comment]
Great, thanks a lot!!!

File: contrib/docker/entrypoint.sh [hunk 1]
```diff
@@ -26,7 +26,15 @@ fi
 chown "${oduser}:${odgroup}" /onedrive/ /onedrive/conf
 
 # Default parameters
-ARGS=(--monitor --verbose --confdir /onedrive/conf --syncdir /onedrive/data)
+ARGS=( --monitor --confdir /onedrive/conf --syncdir /onedrive/data)
```
Comment (abraunegg) [inline contrib/docker/entrypoint.sh]: Need to fix up and remove the additional space here: `( --monitor`

**Comment by abraunegg** [comment]
Once the additional space item is fixed, LGTM, will approve and merge. Many thanks for contributing :)

**Comment by TheFlyingBadger** [comment]
All done.

**Comment by abraunegg** [comment]
@TheFlyingBadger 
Please can you review this patch in light of issue #648 that has been raised. This commit for now has been reverted, and will require a new pull to re-merge.

**Pull request merged**